test/catalog/catalog_test.cpp

```diff
@@ -14,7 +14,7 @@
 #include <unordered_set>
 #include <vector>
 
-#include "buffer/buffer_pool_manager_instance.h"
```

Comment:
I think some of the refactor is not synced?

Reply:
Some of the `BufferPoolManagerInstance` stuff is still present.

Reply:
Fixing now...